src/main/scala/io/moia/protos/teleproto/ReaderImpl.scala

```diff
@@ -426,7 +426,7 @@ object ReaderImpl {
       )
     }
 
-    val surplusModelOptions = modelOptions.filterKeys(name => !protobufOptions.contains(name)).values.map(_.name.decodedName)
+    val surplusModelOptions = modelOptions.filterKeys(name => !protobufOptions.contains(name)).toMap.values.map(_.name.decodedName)
```

Comment:
```suggestion
    val surplusModelOptions = modelOptions
      .filter {
        case (name, _) =>
          !protobufOptions.contains(name)
      }
      .toMap
      .values
      .map(_.name.decodedName)
```

Reply:
Or we could use `filter { case (key, _) => ???` instead which also works

Reply:
Nice, done in c51dc1f